contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -2369,26 +2336,27 @@ class tsqlBuilder : public tsqlCommonMutator
 			}
 		}
 	}
-
+	
 	void exitOrder_by_clause(TSqlParser::Order_by_clauseContext *ctx) override
 	{
 		handleOrderByOffsetFetch(ctx);
 	}
-
-	// NB: this is copied in tsqlMutator
+  
+	// NB: the following are copied in tsqlMutator
 	void exitColumn_def_table_constraints(TSqlParser::Column_def_table_constraintsContext *ctx)
 	{
 		handleTableConstraintWithoutComma(ctx);
 	}
-	
-	// NB: this is copied in tsqlMutator
 	void exitConstant(TSqlParser::ConstantContext *ctx) override
 	{	
 		// Check for floating-point number without exponent
 		handleFloatWithoutExponent(ctx);
 	}
-  
-	// NB: this is copied in TsqlMutator
+	void exitPredicate(TSqlParser::PredicateContext *ctx) override
```

Comment:
It's exactly the same compare to 3201 line

Reply:
see comment above